executor/ddl_test.go

```diff
@@ -676,6 +678,81 @@ func (s *testSuite8) TestShardRowIDBits(c *C) {
 	c.Assert(autoid.ErrAutoincReadFailed.Equal(err), IsTrue, Commentf("err:%v", err))
 }
 
+func (s *testSuite8) TestAutoShardBitsData(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+
+	tk.MustExec("create database if not exists test_auto_shard_bits")
+	defer tk.MustExec("drop database if exists test_auto_shard_bits")
+	tk.MustExec("use test_auto_shard_bits")
+	tk.MustExec("drop table if exists t")
+
+	config.WithAllowAutoShardTemporarySet(true, func() {
+		tk.MustExec("create table t (a bigint primary key auto_shard_bits(15), b int)")
+		for i := 0; i < 100; i++ {
```

Comment:
Please combine those inserts into one(`insert into t (b) values (...), (...)`) for a better performance.